runtime/port/linuxppc/j9ri.c

```diff
@@ -32,6 +32,13 @@
 #define MMCR0_FC        0x80000000
 #define MMCR0_PMAE      0x04000000
 
+/* copied from PPCHWProfilerPrivate.hpp */
```

Comment:
```suggestion
/* Please keep this definition in sync with the one in PPCHWProfilerPrivate.hpp */
```

Reply:
Updated `PPCHWProfilerPrivate.hpp` as well to document this connection.